docs/docs/installation/github.md

```diff
@@ -7,6 +7,7 @@ You can use our pre-built Github Action Docker image to run PR-Agent as a Github
 ```yaml
 on:
   pull_request:
+    types: [opened,reopened,ready_for_review,review_requested]
   issue_comment:
 jobs:
   pr_agent_job:
```

Comment:
/ask can i limit the 'issue_comment' to ignore bots ?  can I limit it to ignore message not starting with '/' ?